agent/run.go

```diff
@@ -14,11 +14,18 @@ const (
 	TaskTypeSink TaskType = "sink"
 )
 
+type RunSink struct {
+	Error   error               `json:"error"`
+	Success bool                `json:"success"`
```

Comment:
do we really need both `success` and `error` here? can we just have either one?
if `error` nil, it must be success or we can just use success only.

Reply:
I see. Even for that I think it is redundant to define success and error altogether. We can just use error and avoid ambiguity for that.

But it is fine I guess for now.